library/spdm_secured_message_lib/libspdm_secmes_encode_decode.c

```diff
@@ -266,6 +266,11 @@ libspdm_return_t libspdm_encode_secured_message(
         LIBSPDM_ASSERT(false);
         return LIBSPDM_STATUS_UNSUPPORTED_CAP;
     }
+
+    /*zero key for security*/
```

Comment:
we need zero key in *any* return path.

Reply:
Thanks. I have added the salt zero in every return path.